Packs/PiHole/Integrations/PiHole/PiHole.py

```diff
@@ -0,0 +1,151 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+''' IMPORTS '''
+
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CONSTANTS '''
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+class Client:
+    def __init__(self, base_url, verify, proxies, auth):
+        self.base_url = base_url
+        self.verify = verify
+        self.proxies = proxies
+        self.auth = '&auth=' + auth
+
+    def http_request(self, method, url_suffix):
```

Comment:
i suggest you use `BaseClient._http_request` instead

Reply:
I would rather not. The coding conventions is following previously advised conventions.